core/geometry/src/test/curve/CurveCurveIntersectXY.test.ts

```diff
@@ -2434,13 +2443,20 @@ describe("CurveCurveIntersectXY", () => {
       Point3d.create(70, 0),
       Matrix3d.createRotationAroundVector(Vector3d.create(0, 0, 1), Angle.createDegrees(90))!,
     );
+    const moveTransform = Transform.createTranslationXYZ(0, 0, 10);
+    const nonPlanarTransform = Transform.createZero();
+    nonPlanarTransform.setMultiplyTransformTransform(rotationTransform, moveTransform);
```

Comment:
Ok, but the new transform just changes the z compared to the other transform.  Z-coordinate is ignored by the xy-intersector. Can you test with a curve that isn't horizontal, or isn't planar?